src/cli.ts

```diff
@@ -12,8 +14,10 @@ declare global {
 }
 
 export function cli(argv: string[], commands: any): void {
+  const rawArgv = argv.slice(2);
+  console.log('RAW ARGV', argv.slice(2));
```

Comment:
did we want to keep this console log in?

Reply:
Removed here: c9b5c21